src/python/ensembl/brc4/runnable/core_server.py

```diff
@@ -88,14 +88,56 @@ def get_cores(self, prefix: str = "", build: str = "", version: str = "") -> Lis
         dbs = self.get_all_cores()
 
         if prefix:
-            dbs = [db for db in dbs if db.startswith(f"{prefix}_")]
+            dbs = [db for db in dbs if db.startswith(f"{prefix}")]
```

Comment:
why so?

Reply:
It's a bit more flexible, think more along the line of selecting databases with their "start" than their prefix. An example: if I create databases with several prefixes like mbarba64a_, mbarba64b_ etc. I will still be able to get the whole list with a prefix="mbarba64".

Reply:
cool. thanks!